content.md

```diff
@@ -236,3 +240,51 @@ Tap on add photo button and select the image to display information that needs t
 ### 6.6 Tap the *CLEAR CACHE* button
 ![screenshot](./images/6/6.jpg)
 <div class="pagebreak"></div>
+
+## 7 How to back-up data to google drive
+
+### 7.1 Start from map view and tap the more icon at top-right of the screen
+![screenshot](./images/map.jpg)
+<div class="pagebreak"></div>
+
+### 7.2 Tap menu icon in the top-right of the screen then select *Saved Forms*
+![screenshot](./images/7/2.jpg)
+<div class="pagebreak"></div>
+
+### 7.3 Tap *BACKUP* in the top-right of the screen
+![screenshot](./images/7/3.jpg)
+<div class="pagebreak"></div>
+
+### 7.4 Tap *Save to drive* in the pop-up menu on the bottom of the screen
+![screenshot](./images/7/4.jpg)
+<div class="pagebreak"></div>
+
+### 7.5 Give a *Document title*, select an *Account* (Goolge) and *Folder* and select save in the bottom-right of the pop-up menu
+![screenshot](./images/7/5.jpg)
+<div class="pagebreak"></div>
+
+### 7.6 a Message should display informing you of the successful upload of the data
+![screenshot](./images/7/6.jpg)
+<div class="pagebreak"></div>
+
+## 8. How to switch on GPS on your device
+
+### 8.1 Start from home screen and swipe down from top of the screen to display the notification drawer
+![screenshot](./images/8/1.jpg)
+<div class="pagebreak"></div>
+
+### 8.2 Swipe down once more to expand this drawer and tap the settings icon (gear)
+![screenshot](./images/8/2.jpg)
+<div class="pagebreak"></div>
+
+### 8.3 In the *Settings* menu under the sub-menu *Personal* tap *Location*
+![screenshot](./images/8/3.jpg)
+<div class="pagebreak"></div>
+
+### 8.4 Tap the toggle bar in the top of the screen to make sure the GPS in *On*, the toggle button should display in green, NOT grey and NOT say *off*. Tap *Mode* in the top-left hand of the screen below the toggle bar
+![screenshot](./images/8/4.jpg)
+<div class="pagebreak"></div>
+
+### 8.5 Tap *High accuracy*, your GPS is now enabled and working on it's optimal functionality
```

Comment:
Change to 'Tap *High accuracy* to ensure that most accurate methods are used to calculate location' and remove the 'GPS is now enabled' part.